lib/perl/Genome/Model/SomaticVariation/Command/DumpIgvXml.pm

```diff
@@ -181,17 +181,11 @@ sub execute {
     #Hardcoded list of allowed reference builds and their mappings to names used in IGV
     my $genome_build = "";
     my $gene_track_name = "";
-    my $starting_locus = "";
-    if ($reference_genome_name eq 'GRCh37-lite-build37'){
-      $genome_build = "b37";
-      $starting_locus = "12:25398182-25398361";
-    }elsif($reference_genome_name eq 'UCSC-mouse-buildmm9'){
-      $genome_build = "mm9";
-      $starting_locus = "12:25398182-25398361";
-    }else{
-      $self->error_message("Unrecognized reference genome name ($reference_genome_name) supplied to DumpIgvXml.pm");
-      return;
-    }
+    my $starting_locus = "12:25398182-25398361";
+    my $genome_build =
+        Genome::Model::Tools::Analysis::ResolveIgvReferenceName::resolve_igv_reference_name($reference_genome_name) ||
```

Comment:
(Same comment about needing to `use Genome::Model::Tools::Analysis::ResolveIgvReferenceName` applies here, too.)